src/components/AvatarOverlayEdit.vue

```diff
@@ -2,12 +2,18 @@
 defineProps<{
   loading: boolean;
   avatar?: string;
+  isViewOnly?: boolean;
 }>();
 </script>
 
 <template>
   <transition name="fade">
     <div
+      v-if="isViewOnly"
```

Comment:
Can we also check if it is a demo site (demo.snapshot.org)? 
It is a recurring support question
https://github.com/snapshot-labs/snapshot/issues/3406

Reply:
![Mar-17-2023 17-53-46](https://user-images.githubusercontent.com/15967809/225903648-e49c9470-8f1b-4af0-af12-0375e2dc3672.gif)
There is no cursor none for the edit button, should be fine?